pkg/scheduler/objects/queue.go

```diff
@@ -657,6 +660,9 @@ func (sq *Queue) addChildQueue(child *Queue) error {
 	if sq.IsDraining() {
 		return fmt.Errorf("cannot add a child queue when queue is marked for deletion: %s", sq.QueuePath)
 	}
+	if sq.maxRunningApps != 0 && sq.maxRunningApps < child.maxRunningApps {
+		return fmt.Errorf("parent maxRunningApps must be larger than child maxRunningApps")
+	}
```

Comment:
This belongs in the config validations check, like we do for the "other" resources. Otherwise we could fail during the config change and leave the system in a strange state.

Reply:
Can you point me where is other resources config validations check?
If you are referring this method: https://github.com/apache/yunikorn-core/blob/branch-1.0/pkg/scheduler/objects/queue.go#L207
I don't think it's a good idea to add this child maxRunningApps check into this method since we need to iterate each child.